test/common_test/Mesh_TEST.cc

```diff
@@ -83,6 +85,81 @@ TEST_F(MeshTest, MeshSubMesh)
   engine->DestroyScene(scene);
 }
 
+/////////////////////////////////////////////////
+TEST_F(MeshTest, MeshSkeleton)
```

Comment:
This is a new test that was added in `ign-rendering6`. I had to modify it to match how we're doing common tests in `gz-rending7`. I'd appreciate more scrutiny here.

Reply:
At least from CI results, seems like there are some issues.

Reply:
There were a few files that I missed to commit. Should hopefully pass now.